docs/design/Liqo-Agent.md

```diff
@@ -0,0 +1,103 @@
+# Liqo Agent
+## Overview
+**Liqo Agent** is the Liqo component that lets the user interact with Liqo in an easy, friendly way.
+It represents one of the two main entry point to the Liqo environment, along with the **LiqoDash**.
+
+_Agent_ is accessible from its tray icon and provides a menu trough which users can:
+
+* monitor the **status** of their own Liqo
+* get **quick access** to Liqo **main features**, e.g. start/stop, Liqo working mode,
```

Comment:
Working mode are not presented in this document, and I feel that we need them at a certain point to make readers understand what it is about.